src/api/integrations/event/webhook/webhook.controller.ts

```
@@ -73,7 +74,16 @@
 
     const webhookConfig = configService.get<Webhook>('WEBHOOK');
     const webhookLocal = instance?.events;
-    const webhookHeaders = instance?.headers;
+    const webhookHeaders = { ...(instance?.headers as Record<string, string> || {}) };
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)